spec/features/admin/members/admin_search_member_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+feature 'Admin can search for users by email' do
+
+  signed_in_as(:admin) do
+    let!(:member) { FactoryGirl.create(:user, :member) }
+
+    before do
+      click_header_option("Dashboard")
+      click_sidemenu_option("Members")
+    end
+
+    scenario "Search with matching data" do
+      fill_in('User email search', :with => member.email)
+      click_button('SEARCH')
+      expect(page).to have_content(member.email)
+      expect(page).not_to have_content(current_user.email)
+    end
+
+    scenario "Search with non-matching data" do
+      fill_in('User email search', :with => current_user.email)
+      click_button('SEARCH')
+      expect(page).to have_content("No users matched that search")
+      expect(page).not_to have_content(current_user.email)
+    end
+  end
+
+end
```

Comment:
I disagree. It is not necessary to have a 29th line for no reason other than a vim reader see it as no eol. Since 99% of work is now done via a text editor in a gui, I don't see the requirement for empty whitespace.